test/hca-rjsf/test-data.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Yeah, I was thinking about it, but in order for this to fill data nested in `view:` containers, they have to also be nested in `view:`. The schema test data expects it to be flattened like it will be when we submit.

Also, this isn't all the fields like we'll want to fill in anyhow; it's required only.